backend/driver.pm

```diff
@@ -54,20 +49,20 @@ sub start {
     my ($self) = @_;
 
     my $p1, my $p2;
-    pipe($p1, $p2) or die "pipe: $!";
+    pipe($p1, $p2) or bmwqemu::logdie("pipe: $!");
     $self->{from_parent} = $p1;
     $self->{to_child}    = $p2;
 
     $p1 = undef;
     $p2 = undef;
-    pipe($p1, $p2) or die "pipe: $!";
+    pipe($p1, $p2) or bmwqemu::logdie("pipe: $!");
     $self->{to_parent}  = $p2;
     $self->{from_child} = $p1;
 
-    printf STDERR "$$: to_child %d, from_child %d\n", fileno($self->{to_child}), fileno($self->{from_child});
+    bmwqemu::fctwarn sprintf "$$: to_child %d, from_child %d\n", fileno($self->{to_child}), fileno($self->{from_child});
 
     my $pid = fork();
-    die "fork failed" unless defined $pid;
+    bmwqemu::logdie "fork failed" unless defined $pid;
```

Comment:
Actually logdie will log and then die, backend developer knows that it died where, and test developer/reviewer knows that he has to nag the backend developer because of this